app/src/main/java/com/example/secondproject/FriendsPageActivity.java

```diff
@@ -47,4 +47,9 @@ public void launchMyProfilePage(View view) {
         Intent myProfileIntent = new Intent(this, MyProfileActivity.class);
         startActivity(myProfileIntent);
     }
+
+    public void launchFriendProfileActivity(View view) {
+        Intent myProfileIntent = new Intent(this, FriendProfileActivity.class);
```

Comment:
Looking good, better than those above, but named it as we agreed about vars name, for example "my_profile_intent".